### PR TITLE
Wofs summaries summary

### PR DESCRIPTION
**Pull request by Kirill888**
plugin to sum wofs annual summaries to compute full history summary.

### CODE REVIEW

File: libs/stats/odc/stats/_wofs.py [hunk 1]
```diff
@@ -114,11 +117,98 @@ def reduce(self, xx: xr.Dataset) -> xr.Dataset:
...
+    @property
+    def measurements(self) -> Tuple[str, ...]:
+        return ("count_wet", "count_clear", "frequency")
+
+    def input_data(self, task: Task) -> xr.Dataset:
+        return dc_load(
+            task.datasets,
+            measurements=["count_wet", "count_clear"],
+            geobox=task.geobox,
+            chunks={},
```
Comment (omad) [inline libs/stats/odc/stats/_wofs.py:156]: What does `chunks={}` do? Activates dask for parallelise loading, but forces it all into a chunk?

**Review by omad — APPROVED** [review]
Looks good to me. I had to look at it for way too long to work out what is was doing and why, so I'd prefer extra documentation. I'll make a PR and you can check my understanding.

Cheers

File: libs/stats/odc/stats/_wofs.py (hunk 1 above)
Comment (Kirill888) [reply libs/stats/odc/stats/_wofs.py:156]: yes single chunk along Y,X but chunk size of 1 for time.

Comment (Kirill888) [reply libs/stats/odc/stats/_wofs.py:156]: so equivalent to `{'x': -1, 'y': -1', 'time': 1}`.

**Pull request merged**